README.md

```diff
@@ -42,7 +42,7 @@ You then need to load the plugin. In `boostrap.php`, something like:
 For a complete setup, add the following to your `App\Controller\AppController`:
 
 ```php
-public $components = ['BootstrapUI.Flash'];
+public $theme = 'BootstrapUI';
```

Comment:
> Doesn't this change mean that one can't use any other theme for their views now?

Why would one need to use other theme? I always viewed the plugin itself a glorified theme.

> Does it also imply that one needs to create their views in a different folder (since they are 'themed')?

No.